src/pages/tutorials/setup/step-two.md

```diff
@@ -113,6 +113,32 @@ Liferay is written in Java, so you will need Java on your machine in order to ha
     ```shell
     java -version
     ```
+5. in your .bash_pro􀃶le we need to make two environment variables: one for JAVA_HOME,
+and one to place that JAVA_HOME variable at the front of your PATH.
+6. to get the JAVA_HOME variable we need to run a line in the terminal.
+    ```shell
+    sudo find/-name javac
```

Comment:
This command seems wrong.  Can you please double check to make sure this is correct?